### ISSUE TITLE
FTP Access

### ISSUE THREAD

**Issue by eubrunosilva** [joel16/CMFileManager-PSP#49]
Hi

I have a PSP 1001 with CFW 6.61 PRO-C

I'm trying to connecto thru FTP and:

- If I connect with WinSCP I dont have any folders to browse.

- If I connect with Filezilla I only have one folder but when I try to go the folder I got this error

Command:	CWD /Jan 0  0000 ms0:
Response:	550 Invalid directory.
Error:	Failed to retrieve directory listing

**Comment by joel16** [comment]
Yeah sometimes the cwd gets dates attached to it. Another thing you can do is to try to manually navigate to "/ms0:" from Filezilla or WinSCP
